### PR TITLE
CORS done

### PR DESCRIPTION
**Pull request by hec7orci7o**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Some ideas for obtaining a :gift: if you are the first that:
 - **Other server is possible**: Uses a web server different from Tomcat and uses a JUnit-based test to prove it.
 - **Compress-me**: Enables HTTP Response compression and uses a JUnit-based test to prove it.
 - **I-like-XML**: Enables content negotiation in `/time` and provides a JUnit-based test to prove it.
-- **CORS what?**: Enables [CORS](https://developer.mozilla.org/es/docs/Web/HTTP/CORS) support and provides a JUnit-based test to prove it.
+- ~**CORS what?**: Enables [CORS](https://developer.mozilla.org/es/docs/Web/HTTP/CORS) support and provides a JUnit-based test to prove it.~ solved by [hec7orci7o](https://github.com/Hec7or-Uni/lab1-git-race).
```
Comment (fjlopez) [inline README.md:11]: Point to your `work` branch in `lab 2`.

**Review by fjlopez — CHANGES_REQUESTED** [review]
`README.md` describes a valid approach to CORS but it is not used in the Controller. Please, update it.

**Review by hec7orci7o — COMMENTED** [review]
work branch linked

**Comment by fjlopez** [comment]
Merged!

**Pull request merged**